backend/src/main/java/gov/cdc/usds/simplereport/api/apiuser/ApiUserDataResolver.java

```diff
@@ -3,33 +3,53 @@
 import gov.cdc.usds.simplereport.api.InternalIdResolver;
 import gov.cdc.usds.simplereport.api.PersonNameResolver;
 import gov.cdc.usds.simplereport.db.model.ApiUser;
-import org.springframework.stereotype.Component;
+import gov.cdc.usds.simplereport.db.model.auxiliary.PersonName;
+import java.util.UUID;
+import org.springframework.graphql.data.method.annotation.SchemaMapping;
+import org.springframework.stereotype.Controller;
 
 /**
  * Resolver for the graphql ApiUser type, which just (currently) contains the name of the person who
  * last modified an entity; this resolver simply aliases nameInfo to name. You may be looking for
  * {@link UserResolver}, which provides access to a much more significant graph.
  */
-@Component
+@Controller
 public class ApiUserDataResolver
     implements InternalIdResolver<ApiUser>, PersonNameResolver<ApiUser> {
+  @SchemaMapping(typeName = "ApiUser", field = "email")
   public String getEmail(ApiUser user) {
     return user.getLoginEmail();
   }
 
+  @SchemaMapping(typeName = "ApiUser", field = "firstName")
   public String getFirstName(ApiUser user) {
     return user.getNameInfo().getFirstName();
   }
 
+  @SchemaMapping(typeName = "ApiUser", field = "middleName")
   public String getMiddleName(ApiUser user) {
     return user.getNameInfo().getMiddleName();
   }
 
+  @SchemaMapping(typeName = "ApiUser", field = "lastName")
   public String getLastName(ApiUser user) {
     return user.getNameInfo().getLastName();
   }
 
+  @SchemaMapping(typeName = "ApiUser", field = "suffix")
   public String getSuffix(ApiUser user) {
     return user.getNameInfo().getSuffix();
   }
+
+  @Override
+  @SchemaMapping(typeName = "ApiUser", field = "id")
+  public UUID getId(ApiUser entity) {
```

Comment:
will have to implement these since you can add the annotation `@SchemaMapping` to a default interface implementation